src/server/game/Battlegrounds/Zones/BattlegroundBE.cpp

```diff
@@ -87,10 +88,10 @@ void BattlegroundBE::HandleAreaTrigger(Player* player, uint32 trigger)
     }
 }
 
-void BattlegroundBE::FillInitialWorldStates(WorldPacket& data)
+void BattlegroundBE::FillInitialWorldStates(WorldPackets::WorldState::InitWorldStates& packet)
 {
-    data << uint32(0x9f3) << uint32(1); // 9
-    Arena::FillInitialWorldStates(data);
+    packet.Worldstates.emplace_back(2547, 1); // BATTLEGROUND_BLADES_EDGE_ARENA_SHOW
```

Comment:
just wondering - any reason to replace the hex numbers by decimals everywhere?

Reply:
no idea why it was changed this way in the cp. I changed it back to hex to reduce the diff